addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -452,6 +461,116 @@ QUnit.test('do not post message with "Enter" keyboard shortcut', async function
     );
 });
 
+QUnit.test("Display suggested recipients list", async function (assert) {
+    assert.expect(10);
+
+    this.data['res.partner'].records.push({ id: 100 });
+    await this.start({
+        async mockRPC(route, args) {
+            if (route === '/mail/get_suggested_recipients') {
+                return Promise.resolve(
+                    {
+                        100: [
+                            [
+                                false,
+                                "john@example.com",
+                                'CC email',
+                            ],
+                            [
+                                false,
+                                "fred@example.com",
+                                'CC email',
+                            ],
+                            [
+                                100,
+                                "second partner",
+                                "CC email"
+                            ],
+                            [
+                                false,
+                                "carl@example.com",
+                                'CC email',
+                            ],
+                        ]
+                    });
+            }
+            return this._super(...arguments);
+        },
+    });
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 100,
+        threadModel: 'res.partner',
+    });
+    await this.createChatterComponent({ chatter });
+
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+
+    assert.containsOnce(
+        document.body,
+        '.o_ComposerSuggestedRecipientList',
+        "Must containt a suggested recipient list"
+    );
+    assert.containsN(
+        document.body,
+        '.o_ComposerSuggestedRecipient',
+        3,
+        "at start, the list should not contain more than 3 suggested recipients"
```

Comment:
_"There should be 3 suggested recipients"_

_(not necessary to put "initially" or "after click on 'Send message' button", because the previous assertion comment is clear enough, so on QUnit UI context is good)_